src/Symfony/Component/Runtime/Internal/ComposerPlugin.php

```diff
@@ -74,7 +74,7 @@ public function updateAutoloadFile(): void
             }
         }
 
-        $projectDir = $fs->makePathRelative($projectDir, $vendorDir);
+        $projectDir = $fs->makePathRelative(realpath($projectDir.($extra['project_dir'] ?? '')), $vendorDir);
```

Comment:
doesn't this miss a `/`?
```suggestion
        $projectDir = $fs->makePathRelative(realpath($projectDir.'/'.($extra['project_dir'] ?? '')), $vendorDir);
```

Reply:
yes, that makes sense

Reply:
did you forgot this comment?

Reply:
oops, my bad: done